lib/registries/addon/overview/template.hbs

```diff
@@ -138,6 +138,44 @@
                     <leftNav.link data-analytics-name='Comments' @route='registries.overview.comments'
                         @models={{array this.registration.id}} @icon='comments'
                         @label={{t 'registries.overview.comments.title'}} @count={{this.commentsCount}} />
+
+                    <div data-test-resources-wrapper local-class='ResourcesWrapper'>
+                        <div local-class='ResourcesWrapper_title'>
+                            <h3>{{t 'registries.overview.resources.title'}}</h3>
+                            <FaIcon local-class='HelpIcon' @icon='question-circle' />
+                            <BsTooltip
+                                @placement='right'
+                                @triggerEvents='hover'
+                            >
+                                {{t 'registries.overview.resources.title'}}
+                            </BsTooltip>
```

Comment:
I don't believe the mockups called for a help icon here.

Reply:
Hey Brian, here's what I have for the current mock-ups:

<img width="1673" alt="image" src="https://user-images.githubusercontent.com/82047646/183431399-df3ec07e-5434-4971-9e92-6cc51e53d883.png">

We do not currently have guidance for the BsTooltip hover text, but during our earlier discussion you had suggested using 'Open Resources' as a placeholder. Does this still work?